### ISSUE TITLE
Hosting the Mod Template on Github

### ISSUE THREAD

**Issue by MrMonotone** [jmcmorris/CreaDocs#6]
I think that it would be useful to host the mod template on Github. 
Reasons to host on Github:
* Always up to date
* Easy to update
* Introduces users to github
People can just download the repo and work with it without git. There may be a better way to do this. However I have hosted the template [here](https://github.com/MrMonotone/crea_mod_template).

**Comment by jmcmorris** [comment]
Agreed! Although it might be nice to have the mod template on github with examples? The original idea for the mod template is mostly for the directory setup so modders don't have to worry about that and hopefully keep things consistent.
